### PR TITLE
style(locale): Translate to French  Batch 1

### PR DESCRIPTION
**Pull request by UchiTesting**
Initial batch to the French translation.

Items translated

- [x] Main window (when no repo is open)
  - [x] Text in welcome tab (listing known repos)
  - [x] Text in the search bar
  - Repo tree
    - [x] Context menu for a group
    - [x] Context menu for a repo
    - [x] Note in the bottom of the window
- [x] Tabs bar
  - [x] Tooltip: Welcome tab
  - [x] Menu contextuel de l'tab d'accueil
- [x] Barre d'outils principale
  - [x] Tooltip: Clone a repo
  - [x] Tooltip: Open a repo
  - [x] Tooltip: Open terminal
  - [x] Tooltip: Create a group
- [x] Main menu
  - [x] Text for  Preferences
    - [x] tab General
    - [x] tab Appearence
    - [x] tab GIT
    - [x] tab GPG Signing
    - [x] tab DIFF/Merge tool
  - [x] Text for Menu Open AppData folder
  - [x] Text for Menu Keyboard shortcut reference
    - [x] Texts in the dialog
  - [x] Text for Menu Check update
    - [x] A set of texts related to the feature
  - [x] Text for Menu About
    - [x] Dialog About
- [x] Dialog Clone repo
- [X] Dialog OpenRepo (not needed/ provided by framework/OS)
- [x] Dialog d'ouverture d'un terminal (not needed/ provided by framework/OS)

### CODE REVIEW

**Comment by UchiTesting** [comment]
Rebased to develop branch.
Should be OK.

image

File: src/Resources/Locales/fr_FR.axaml [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+  <x:String x:Key="Text.CopyFileName" xml:space="preserve">Copier le nom de fichier</x:String>
+  <x:String x:Key="Text.CreateBranch" xml:space="preserve">Créer une branche...</x:String>
+  <x:String x:Key="Text.CreateBranch.BasedOn" xml:space="preserve">Basé sur :</x:String>
+  <!-- A vérifier si féminin-->
+  <x:String x:Key="Text.CreateBranch.Checkout" xml:space="preserve">Check out the created branch</x:String>
+  <x:String x:Key="Text.CreateBranch.LocalChanges" xml:space="preserve">Changements locaux :</x:String>
+  <x:String x:Key="Text.CreateBranch.LocalChanges.Discard" xml:space="preserve">Rejeter</x:String>
+  <x:String x:Key="Text.CreateBranch.LocalChanges.DoNothing" xml:space="preserve">Ne rien faire</x:String>
+  <x:String x:Key="Text.CreateBranch.LocalChanges.StashAndReply" xml:space="preserve">Stash &amp; Réappliquer</x:String>
+  <x:String x:Key="Text.CreateBranch.Name" xml:space="preserve">Nouveau non de branche:</x:String>
```
Comment (Jayllyz) [inline src/Resources/Locales/fr_FR.axaml:162]:
```suggestion
  <x:String x:Key="Text.CreateBranch.Name" xml:space="preserve">Nouveau nom de branche:</x:String>
```

File: src/Resources/Locales/fr_FR.axaml [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+  <!-- A vérifier si féminin-->
+  <x:String x:Key="Text.CreateTag.Type.Lightweight" xml:space="preserve">lightweight</x:String>
+  <x:String x:Key="Text.CtrlClickTip" xml:space="preserve">Maintenir Ctrl pour commencer directement</x:String>
+  <x:String x:Key="Text.Cut" xml:space="preserve">Couper</x:String>
+  <x:String x:Key="Text.DeleteBranch" xml:space="preserve">Supprimer la branche</x:String>
+  <x:String x:Key="Text.DeleteBranch.Branch" xml:space="preserve">Branche :</x:String>
+  <x:String x:Key="Text.DeleteBranch.IsRemoteTip" xml:space="preserve">Vous êtes sur le point de supprimer une branche distante !!!</x:String>
+  <x:String x:Key="Text.DeleteBranch.WithTrackingRemote" xml:space="preserve">Supprimer également la branche distante ${0}$</x:String>
+  <x:String x:Key="Text.DeleteMultiBranch" xml:space="preserve">Supprimer plusieurs branches</x:String>
+  <x:String x:Key="Text.DeleteMultiBranch.Tip" xml:space="preserve">Vous essayez de supprimer plusieurs branche à la fois. Assurez-vous de revérifier avant de procéder !</x:String>
```
Comment (Jayllyz) [inline src/Resources/Locales/fr_FR.axaml:185]:
```suggestion
  <x:String x:Key="Text.DeleteMultiBranch.Tip" xml:space="preserve">Vous essayez de supprimer plusieurs branches à la fois. Assurez-vous de revérifier avant de procéder !</x:String>
```

**Comment by love-linger** [comment]
Thank you very much. @UchiTesting @Jayllyz.

**Pull request merged**